configure

```diff
@@ -7,7 +7,7 @@
 
 import sys
 if not sys.version_info >= (3, 4):
-    print("openage requires Python 3.4 or higher")
+    print("Openage requires Python 3.4 or higher.")
```

Comment:
Things like iPad are the exception, but you would capitalize Firefox and Steam. Stack Overflow is being inconsistent, as they are using "stackoverflow" for their logo, and yet their site is called "Stack Overflow". It's all over the place. :P